src/arduino/7_segment_display/7_segment_display.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 7-Segment Display
```

Comment:
Har du noe forslag? Vi fant ikke noe mer norsk enn det her, hvor jo eneste fremmedordet for så vidt er display.